### DISCUSSION TITLE
[Script request] OSX-KVM or LXC

### DISCUSSION THREAD

**Discussion by chrisjenx** [community-scripts/ProxmoxVE#99] [Request script] [answered]
### Application Name

OSx

### Website

https://github.com/sickcodes/Docker-OSX

### Description

Maybe a little controversial, but we build iOS apps, and a fleet of Mac Mini's get expensive...

There are a couple of approchs, not sure if it's possible to use the docker image directly in lxc? https://github.com/sickcodes/Docker-OSX

But there is https://www.nicksherlock.com/2022/10/installing-macos-13-ventura-on-proxmox/#more-1349 and https://github.com/thenickdude/OSX-KVM

Understandable if this is a bit more involved than you would like

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
We had this topic at work a few weeks ago. Even with docker it didn't run cleanly. Therefore currently no need for a script, which sucks to maintain

**Reply by chrisjenx** [reply]
Dang that's a bummer, wanted an easy way to build our iOS apps without an army of bloody Mac Minis
